src/platform/pdf/models/create-from-html-args.dto.ts

```diff
@@ -0,0 +1,4 @@
+export class CreateFromHtmlArgsDto {
+  signedUrl: string;
```

Comment:
Are we sending a signedUrl to html file located in an S3 bucket? Is there any reason we're not sending the HTML straight away in the call? 
Also: The input type and lack of comments might be confusing for the developer calling this.